source/includes/auth.md

```diff
@@ -35,38 +50,101 @@ The `sessionId` token should be passed with all subsequent calls as a cookie nam
 
 ## Logout User/End session
 
-```console
+```http
 DELETE /openmrs/ws/rest/v1/session -H 'Accept: application/json'
--H 'Authorization: Basic Auth' (required to indetify the user)
+-H 'Authorization: Basic Auth' (required to identify the user)
+```
+
+```java
+	OkHttpClient client = new OkHttpClient().newBuilder().build();
+	Request request = new Request.Builder()
+		.url("https://demo.openmrs.org/openmrs/ws/rest/v1/session")
+		.method("DELETE", null)
+		.addHeader("Authorization", "Basic YWRtaW46QWRtaW4xMjM=")
+		.build();
+	Response response = client.newCall(request).execute();		
 ```
 
+
+
 ## Changing Password
-```console
+```java
+
+	OkHttpClient client = new OkHttpClient().newBuilder()
+			.build();
+	MediaType mediaType = MediaType.parse("application/json");
+	
+	//password should contain atleast 1 integer
+	RequestBody body = RequestBody.create(mediaType, "{ \r\n    \"newPassword\" : \"password1\"\r\n}");
+	Request request = new Request.Builder()
+			.url("https://demo.openmrs.org/openmrs/ws/rest/v1/password/45ce6c2e-dd5a-11e6-9d9c-0242ac150002")
+			.method("POST", body)
+			.addHeader("Authorization", "Basic YWRtaW46QWRtaW4xMjM=")
+			.addHeader("Content-Type", "application/json")
+			.build();
+	Response response = client.newCall(request).execute();
+}
+
+```
+```http
 POST /openmrs/ws/rest/v1/password/:target_user_uuid 
 {
   "newPassword" : "newPassword"
 }
 ``` 
-```console
+
+
+
+<b>Since version 2.17 of the webservices.rest module:</b>
+
+* An administrator (with the `EDIT_USER_PASSWORDS` privilege) can change the password for other users by 
+  posting a new password to `/password/:target_user_uuid`.
+* The new password must contain atleast one integer.
+
+  
+```http
```

Comment:
add console back

Reply:
@ayesh if I add console back its not a supported language in slate so it will appear across all the panes even in java pane as well , therefore I changed it to http